CMakeLists.txt

```diff
@@ -27,10 +27,10 @@ set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 find_package(CLANG_FORMAT 10)
 find_package(MPI REQUIRED)
 find_package(Kokkos REQUIRED)
-find_package(Cabana REQUIRED COMPONENTS Cabana::Cajita Cabana::cabanacore)
 find_package(HYPRE REQUIRED)
-find_package(ArborX REQUIRED)
 find_package(Boost 1.66.0 REQUIRED)
+find_package(ArborX REQUIRED)
+find_package(Cabana REQUIRED COMPONENTS Cabana::Cajita Cabana::cabanacore)
```

Comment:
Did you mean to commit these changes?

Reply:
yes we were surprisingly getting configure errors with HYPRE if they weren't in this order.

Reply:
I opened #40 - hopefully that works for you as well